app/src/constants.ts

```diff
@@ -72,3 +72,11 @@ export const CROP_DELAY_MS = 11000
 
 // Date time constants
 export const ACCOUNT_EXPIRATION_DATE_FORMAT = 'MMMM D, YYYY'
+
+// BCSC Vidoe Resolution
+export const SD_VIDEO = { width: 640, height: 480 } // standard definition video resolution
+export const SELFIE_FRAME_RATE = 24
+export const MAX_SELFIE_VIDEO_DURATION_SECONDS = 30
+
+// File chunking constants
+export const DEFAULT_CHUNK_SIZE = 1024 * 1024 // 1 MB
```

Comment:
question: Should we use a smaller default? ie: 64kb (64 * 1024)

Curious what you think from your exploration with react-native-fs package and general related reading for this PR.